src/bundle.c

```diff
@@ -1058,13 +1058,190 @@ static struct list *get_dir_files_sorted(char *path)
 	return list_sort(files, lex_sort);
 }
 
+struct alias_lookup {
+	char *alias;
+	struct list *bundles;
+};
+
+/* Deep free on alias_lookup structs
+ */
+static void free_alias_lookup(void *lookup)
+{
+	struct alias_lookup *l = (struct alias_lookup *)lookup;
+	free(l->alias);
+	list_free_list_and_data(l->bundles, free);
+	free(l);
+}
+
+/* Get bundles for a given alias but if the alias is
+ * not actually an alias, return a list with just the
+ * bundle name
+ */
+static struct list *get_alias_bundles(struct list *alias_definitions, char *alias)
+{
+	struct list *bundles = NULL;
+	struct list *iter = alias_definitions;
+
+	while (iter) {
+		struct alias_lookup *lookup = (struct alias_lookup *)iter->data;
+		iter = iter->next;
+		if (strcmp(lookup->alias, alias) == 0) {
+			bundles = list_deep_clone_strs(lookup->bundles);
+			break;
+		}
+	}
+	if (!bundles) {
+		bundles = list_prepend_data(bundles, strdup_or_die(alias));
+	}
+
+	return bundles;
+}
+
+/* Parse an alias definition file for a list of bundles
```

Comment:
Please, document too the order of override. And we need to document that in the manpages too in doc/ folder

Reply:
Will do.

Reply:
Do you want me to add an rst.in file to get the paths for system and user alias directories into the man pages?

Reply:
I attempted to do this, hopefully everything looks reasonable now.